_common.R

```diff
@@ -14,7 +14,7 @@ options(dplyr.print_min = 6, dplyr.print_max = 6)
 
 # Activate crayon output
 options(
-  crayon.enabled = TRUE,
+  #crayon.enabled = TRUE,
```

Comment:
Can you make sure to file an issue in this repo to track re-enabling this?

Reply:
Done #1027